cli/dcoscli/data/help/dcos.txt

```diff
@@ -1,39 +1,42 @@
-Command line utility for the Mesosphere Datacenter Operating
-System (DCOS)
-
-'dcos help' lists all available subcommands. See 'dcos <command> --help'
-to read about a specific subcommand.
+Description:
+    The Mesosphere Datacenter Operating System (DCOS) spans all of the machines in
+your datacenter or cloud and treats them as a single, shared set of resources.
 
 Usage:
-    dcos [options] [<command>] [<args>...]
+     dcos [options] [<command>] [<args>...]
 
 Options:
-    --help                      Show this screen
-    --version                   Show version
-    --log-level=<log-level>     If set then print supplementary messages to
-                                stderr at or above this level. The severity
-                                levels in the order of severity are: debug,
-                                info, warning, error, and critical. E.g.
-                                Setting the option to warning will print
-                                warning, error and critical messages to stderr.
-                                Note: that this does not affect the output sent
-                                to stdout by the command.
-    --debug                     If set then enable further debug messages which
-                                are sent to stdout.
-
-Environment Variables:
-    DCOS_LOG_LEVEL              If set then it specifies that message should be
-                                printed to stderr at or above this level. See
-                                the --log-level option for details.
+    --debug
+        Enable debug mode
+    --help
+        Print usage
+    --log-level=<log-level>
+        Set the logging level
+    --version
+        Print version information
 
-    DCOS_CONFIG                 This environment variable points to the
-                                location of the DCOS configuration file.
-                                [default: ~/.dcos/dcos.toml]
+Positional Arguments:
+    <log-level>
+        The severity level:
+        * debug    Prints all messages.
+        * info     Prints informational, warning, error, and critical messages.
+        * warning  Prints warning, error, and critical messages.
+        * error    Prints error and critical messages.
+        * critical Prints only critical messages to stderr.
+        This setting does not affect the output sent to stdout.
```

Comment:
This doesn't look right. I think we need to move this up to the `--log-level` entry in line 13.